examples/daal/cpp_sycl/makefile_lnx

```diff
@@ -100,7 +100,7 @@ COPTS := -pedantic -Wall -Wextra -Werror \
          -fsycl-device-code-split=per_kernel \
          -I./source/utils 
 
-LOPTS := -Wl,--start-group -Wl,$(DAAL_LIB),$(EXT_LIB) $(DAAL_PATH)/libonedal_sycl.a -Wl,--end-group
+LOPTS := -Wl,--start-group -Wl,$(DAAL_LIB),$(EXT_LIB) $(DAAL_PATH)/libonedal_sycl.a $(DAAL_PATH)/libonedal_core.a -Wl,--end-group
```

Comment:
Yes, thank you for pointing that out. I created new commit to address this. The reason I had this is because core.a contains reference needed for sycl.a undefined references, but so does dpc.so. I add in a conditional to link dpc if dynamic since core will be linked for static already.